python/tidy/servo_tidy/tidy.py

```diff
@@ -694,6 +695,36 @@ def check_spec(file_name, lines):
                 brace_count -= 1
 
 
+def check_config_file(lines):
+    current_table = ""
+    for idx, line in enumerate(lines):
```

Comment:
Since we've planned to prioritize this check, instead of relying on `collect_errors_for_files`, we should initially check whether the config file exists, get the contents, and begin tidying.